pages/main.page.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from "@playwright/test";
+
+export default class MainPage {
+
+    constructor(public page: Page) {}
+
+    bannersLocator = '.blocks-promo';
+    logoLocator = '.logo';
+
+    async checkBannersVisible(){
```

Comment:
```suggestion
    async verifyBannersAreVisible(){
```
лучше использовать verify и вспомогательный глагол, но это уже субъективно)